### PR TITLE
Add missing subdirectories after refactor/move of components

### PR DESCRIPTION
**Pull request by smeyn**
the app component was moved into the app/main component and a new component 'app/terms' was added.
Due to a faulty .gitignore this was not picked up. Now fixed up.

### CODE REVIEW

File: tools/bq-visualizer/src/app/main/main.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (ryanmcdowell) [inline tools/bq-visualizer/src/app/main/main.component.spec.ts]: Missing license header.

File: tools/bq-visualizer/src/app/main/main.component.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.main {
```
Comment (ryanmcdowell) [inline tools/bq-visualizer/src/app/main/main.component.css]: Missing license header

File: tools/bq-visualizer/src/app/terms/terms.component.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.view {
```
Comment (ryanmcdowell) [inline tools/bq-visualizer/src/app/terms/terms.component.css]: Missing license header

File: tools/bq-visualizer/src/app/terms/terms.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (ryanmcdowell) [inline tools/bq-visualizer/src/app/terms/terms.component.spec.ts]: Missing license header

File: tools/bq-visualizer/src/app/terms/terms.component.ts [hunk 5]
```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2018 Google LLC
```
Comment (ryanmcdowell) [inline tools/bq-visualizer/src/app/terms/terms.component.ts]: Different copyright year than other files.

File: tools/bq-visualizer/src/app/main/main.component.css (hunk 2 above)
Comment (smeyn) [reply tools/bq-visualizer/src/app/main/main.component.css]: done

File: tools/bq-visualizer/src/app/terms/terms.component.css (hunk 3 above)
Comment (smeyn) [reply tools/bq-visualizer/src/app/terms/terms.component.css]: done

File: tools/bq-visualizer/src/app/terms/terms.component.spec.ts (hunk 4 above)
Comment (smeyn) [reply tools/bq-visualizer/src/app/terms/terms.component.spec.ts]: done

File: tools/bq-visualizer/src/app/terms/terms.component.ts (hunk 5 above)
Comment (smeyn) [reply tools/bq-visualizer/src/app/terms/terms.component.ts]: done

**Pull request merged**